homeassistant/components/growatt_server/sensor.py

```diff
@@ -80,7 +80,12 @@ async def async_setup_entry(
         config[CONF_URL] = url
         hass.config_entries.async_update_entry(config_entry, data=config)
 
-    api = growattServer.GrowattApi()
+    # Initialise the library with a random user id each time it is started, also extend the library's default identifier to include 'home-assistant'
+    api = growattServer.GrowattApi(
+        add_random_user_id=True,
+        agent_identifier=growattServer.GrowattApi.agent_identifier
+        + " - home-assistant",
```

Comment:
Please use f-strings instead of string concatenation.

Reply:
Done